lnrpc/lightning.proto

```diff
@@ -598,6 +598,21 @@ service Lightning {
     */
     rpc LookupHtlcResolution (LookupHtlcResolutionRequest)
         returns (LookupHtlcResolutionResponse);
+
+    /*
+    Quiesce instructs a channel to initiate the quiescence (stfu) protocol. This
+    RPC is for testing purposes only. The commit that adds it will be removed
+    once interop is confirmed.
+    */
+    rpc Quiesce (QuiescenceRequest) returns (QuiescenceResponse);
```

Comment:
Can move this to [devrpc](https://github.com/lightningnetwork/lnd/tree/fcde6ce2726628a1d0824ef7f7f88f7cd4fc905c/lnrpc/devrpc), which needs a special build tag, and is active by default in the itests.